test/e2e/utils/kubectl.go

```diff
@@ -63,29 +63,26 @@ func (k *Kubectl) Apply(inNamespace bool, cmdOptions ...string) (string, error)
 	ops := append([]string{"apply"}, cmdOptions...)
 	if inNamespace {
 		return k.CommandInNamespace(ops...)
-	} else {
-		return k.Command(ops...)
 	}
+	return k.Command(ops...)
```

Comment:
What is the issue with the previous code? (There are two changes below that are exactly the same)

Reply:
See: https://github.com/kubernetes-sigs/kubebuilder/pull/1291#discussion_r365094307

Reply:
Marking as resolved since it is duplicated of another comment.